cmd/kube-apiserver/app/server.go

```diff
@@ -654,16 +649,12 @@ func buildServiceResolver(enabledAggregatorRouting bool, hostname string, inform
 		return testServiceResolver
 	}
 
-	var serviceResolver webhook.ServiceResolver
+	serviceResolver := aggregatorapiserver.NewClusterIPServiceResolver(informer.Core().V1().Services().Lister())
```

Comment:
I think the existing code is more correct... rather than constructing and then overriding an implementation, we only construct the implementation we actually want